pkg/certificate/service.go

```diff
@@ -209,6 +260,11 @@ func New(ctx context.Context, idCfg *config.IdentityConfig) (daemon.Daemon, erro
 	}
 
 	run := func() error {
+		err := isValidFiles()
+		if err != nil {
+			return err
+		}
```

Comment:
I am not even sure if you should run this every time you insert this. I think its okay cuz it only inserts every 24 hours, but again check with fsudo san (and other teammates) whether running this only once during the init phase is okay enough or not.

Reply:
And again, write it one liner without defining them each

```suggestion
  if err := isValidFiles(); err != nil { ... }
```

Reply:
just realized the name `isValidFiles` is grammatically incorrect. should be `areValidFiles()`
And also you don't have to start with `isXXX` or `areXXX` all the time, where lots of full stack app do so but not in go lang.

The following is much easier to read 
```
  if err := validateFiles() { ... }
```

Reply:
> I am not even sure if you should run this every time you insert this. I think its okay cuz it only inserts every 24 hours, but again check with fsudo san (and other teammates) whether running this only once during the init phase is okay enough or not.

I think it is necessary to run this every time.
The file status may change within 24 hours, so I think this verification logic is necessary before writing.